package.json

```diff
@@ -6,12 +6,13 @@
   "repository": "https://github.com/suchipi/chai-jest-snapshot",
   "scripts": {
     "build": "rm -rf dist/* && babel src --out-dir dist",
-    "mocha": "mocha -r 'dist/spec/index' 'dist/**/*.spec.js'",
+    "jest": "jest --roots dist/spec-jest",
+    "mocha": "mocha -r 'dist/spec/index' 'dist/spec/*.spec.js'",
     "examples": "cd examples && npm test",
     "example_1:watch": "npm run build && cd examples && npm run example_1:watch",
     "example_2:watch": "npm run build && cd examples && npm run example_2:watch",
     "example_3:watch": "npm run build && cd examples && npm run example_3:watch",
-    "test": "npm run build && npm run mocha && npm run examples"
+    "test": "npm run build && npm run mocha && npm run jest && npm run examples"
```

Comment:
We don't need to run jest here at the top level because it gets run by the examples, so let's remove it from the test script.